data/transit/route/bus.json

```diff
@@ -3796,16 +3796,21 @@
       }
     },
     {
-      "displayName": "CTS",
+      "displayName": "CTS (Compagnie des Transports Strasbourgeois)",
       "id": "cts-f1d622",
-      "locationSet": {"include": ["de", "fx"]},
+      "locationSet": {"include": ["de", "fr"]},
```

Comment:
Sorry, I thought that it was an error, I didn't understand what "fx" meant. I just fixed that with a new commit.
However, I removed the german location for bus routes, since there is no bus line operated by the CTS in germany (but one tram line).